lib/ui/community/LinkBox.dart

```diff
@@ -0,0 +1,130 @@
+import 'package:flutter/material.dart';
+
+class _Constants {
+  static final Color cardBackgroundColor = Color(0xfff5f8fa);
```

Comment:
this should be in the style

Reply:
we don't want all of the linkboxtohave what's app´s green color 🤦‍♂

Reply:
I think that it shouldn't have viewmodel and style. It's an atom for me. But yes, I will pass a background color and icon color as a param to make it more dynamic

Reply:
I am agree with @DavidFD005 in this case, I see that as an Atom as it does contain only Flutter Widgets. But yes, we should need to have something to send these values to the Widget to style it.

Reply:
finally I've made it like a molecule, because the list of params became to large.

Reply:
Images updated